### PR TITLE
[Fix] Open web views in 3/4 in the PaymentsHubFragment

### LINKED ISSUE #12625: [Mobile Payments] Inconsistencies in the way we open a web view
**Describe the bug**
I noticed inconsistency in the way we display web view when clicked on any of the “learn more” links in the payments settings. The web view in “Manage Card Reader” and “What is Tap to Pay?” opens partially (which is expected), while the web view from Payment Settings opens in full screen.

**To Reproduce**
1. Navigate to payments settings (More menu -> Payments)
2. Click on "Manage card reader" or "What is Tap to Pay" option
3. Click on the "Learn more" link and see web view opens in half initially (Expected)
4. Now, come back to the payment settings, scroll down and click on "Learn more"
5. See that the web view opens in a full screen.

https://github.com/user-attachments/assets/8945b888-b6dd-4283-8b80-9919e9720618

**Expected behavior**
Web view should open in half on every screens.

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #12625 

### Description

The PR makes all the web views to be open in 3/4 of the screen on, to make it consistent

### Steps to reproduce

More -> Payments -> Learn more

### The tests that have been performed

Only STR

### Images/gif

https://github.com/user-attachments/assets/d5b1d05a-5552-4a96-a80f-0165aac92957

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

625

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/243">21.1</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/hub/PaymentsHubFragment.kt [hunk 1]
```diff
@@ -123,7 +123,11 @@ class PaymentsHubFragment : BaseFragment(R.layout.fragment_payments_hub) {
                     )
                 }
                 is PaymentsHubViewModel.PaymentsHubEvents.OpenGenericWebView -> {
-                    ChromeCustomTabUtils.launchUrl(requireContext(), event.url)
+                    ChromeCustomTabUtils.launchUrl(
+                        context = requireContext(),
+                        url = event.url,
+                        height = ChromeCustomTabUtils.Height.Partial.ThreeQuarters,
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/hub/PaymentsHubFragment.kt:129]: This means that web view will be in 3/4 also for:
* TTP requirements
* Learn more about cash on delivery
Do you think it's ok?

Comment (AnirudhBhat) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/hub/PaymentsHubFragment.kt:129]: This shouldn't be a problem. IMO, most of the Android users are familiar that they can swipe the view up (web view) if they want to read further. WDYT?

**Review by AnirudhBhat — APPROVED** [review]
Thanks for working on this. LGTM :shipit:

**Comment by kidinov** [comment]
@AnirudhBhat 

> This shouldn't be a problem. IMO, most of the Android users are familiar that they can swipe the view up (web view) if they want to read further. WDYT?

I think this is fine

Could you also go through the check list on the PR description?

**Pull request merged**